misc/network.c

```diff
@@ -1001,7 +1001,7 @@ open_stream_connection_ext (struct script_infos *args, unsigned int port,
 
     default:
       g_message ("open_stream_connection_ext(): unsupported transport"
-                 " layer %d", transport);
+                 " layer %d executing %s", transport, args->name);
```

Comment:
As the "transport" layer variable is passed by the nvt and not executed by the NVT maybe replace

"executing"

with

"passed by". Besides this the change looks fine to me.